presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestDetermineJoinDistributionType.java

```diff
@@ -246,6 +248,144 @@ public void testRetainDistributionType()
                 .doesNotFire();
     }
 
+    @Test
+    public void testHighConfidenceLeftAndLowConfidenceRight()
+    {
+        int aRows = 50;
+        int bRows = 30;
+        assertDetermineJoinDistributionType()
+                .setSystemProperty(JOIN_DISTRIBUTION_TYPE, JoinDistributionType.AUTOMATIC.name())
+                .setSystemProperty(CONFIDENCE_BASED_BROADCASTING_ENABLED, "TRUE")
+                .overrideStats("valuesA", PlanNodeStatsEstimate.builder()
+                        .setConfidence(HIGH)
+                        .setOutputRowCount(aRows)
+                        .addVariableStatistics(ImmutableMap.of(new VariableReferenceExpression(Optional.empty(), "A1", BIGINT), new VariableStatsEstimate(0, 100, 0, 6400, 100)))
+                        .build())
+                .overrideStats("valuesB", PlanNodeStatsEstimate.builder()
+                        .setOutputRowCount(bRows)
```

Comment:
I didn't see the setConfidence() here, I guess the default is LOW?

Reply:
@feilong-liu Yes, default confidence is LOW, but to make it more clear I can set the confidence as LOW